### PR TITLE
bpo-35661: Store the venv prompt in pyvenv.cfg

### PR DESCRIPTION
**Pull request by csabella**
https://bugs.python.org/issue35661

### CODE REVIEW

**Comment by remilapeyre** [comment]
This looks good to me.

**Review by remilapeyre — APPROVED** [review]

File: Lib/venv/__init__.py [hunk 1]
```diff
@@ -158,6 +158,7 @@ def create_configuration(self, context):
                 incl = 'false'
             f.write('include-system-site-packages = %s\n' % incl)
             f.write('version = %d.%d.%d\n' % sys.version_info[:3])
+            f.write(f'prompt = {context.prompt}\n')
```
Comment (zooba) [inline Lib/venv/__init__.py]: Dealing with leading/trailing whitespace could get interesting here. All the other values can be trimmed at both ends, while this one (currently) must have one space removed from the start.

Perhaps we should insert the `repr` of the prompt instead? Or at least double-quote it and escape internal quotes and backslashes?

Comment (csabella) [reply Lib/venv/__init__.py]: Thanks for the suggestion.

**Review by brettcannon — APPROVED** [review]

**Comment by brettcannon** [comment]
Thanks for doing this, @csabella !

**Comment by brettcannon** [comment]
@csabella did you want to go ahead and merge this yourself? Or did you need anything to help finish this PR?

**Comment by csabella** [comment]
@brettcannon Thanks!  I'm still not always sure when I should merge or not, so I appreciate that you asked.  :-)

**Pull request merged**